src/functionality/configUtils.ts

```diff
@@ -53,3 +53,20 @@ export function getSharedTheme(
 
   return sharedTheme;
 }
+
+//----------------------------------
+//  updateViewsCalciteMode
+//----------------------------------
+
+/**
+ * Update view's calcite-mode class when theme is updated
+ */
+
+export function updateViewsCalciteMode(mode: "light" | "dark"): void {
+  const prevCalciteMode = mode === "light" ? "calcite-mode-dark" : "calcite-mode-light";
+  const calciteMode = mode === "light" ? "calcite-mode-light" : "calcite-mode-dark";
+  const uiContainers = document.querySelectorAll(`.esri-ui.${prevCalciteMode}`);
```

Comment:
optional - if we're querying for elements from the view, it may be more optimal to use `querySelectorAll` from the view node rather than `document`.

Reply:
I did it this way because of Exhibit, depending on the setting could have 4 different views. This would require 4 watchers and have to run 4 times if it was just on the view instead of just getting it all at once with document.